src/status_im/ui/screens/wallet/main/views.cljs

```diff
@@ -72,20 +73,46 @@
     [rn/view
      [asset-list-item row]]]))
 
-(defn asset-section []
-  (let [assets {"eth" {:currency :eth :amount 0.445}
-                "snt" {:currency :snt :amount 1}
-                "gno" {:currency :gno :amount 0.024794}}]
+(def assets-example-map
+  {"eth" {:currency :eth :amount 0.445}
+   "snt" {:currency :snt :amount 1}
+   "gno" {:currency :gno :amount 0.024794}})
+
+;; NOTE(oskarth): In development, replace assets with assets-example-map
+;; to check multiple assets being rendered
+(defn asset-section [eth]
+  (let [assets {"eth" {:currency :eth :amount eth}}]
     [rn/view {:style st/asset-section}
      [rn/text {:style st/asset-section-title} "Assets"]
      [rn/list-view {:dataSource      (lw/to-datasource assets)
                     :renderSeparator (when platform/ios? (render-separator-fn (count assets)))
                     :renderRow       render-row-fn}]]))
 
+(defn wallet->eth-balance [wallet]
+  (when-let [balance (:balance wallet)]
+    (money/wei->ether balance)))
+
+(defn wallet+prices->portfolio-value [{:keys [balance]} {:keys [price]}]
```

Comment:
Let's call it `portfolio-value`. `wallet` and `prices` are parameters, when we call it it looks weird `(wallet+prices->portfolio-value wallet prices)`